### PR TITLE
adding a new profile

### PR DESCRIPTION
**Pull request by dunyanong**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1270"></a>

### CODE REVIEW

File: public/data/dunyanong.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        "url": "https://github.com/dunyanong",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/dun_yan_",
+        "icon": "twitter"
+      }
+    ]
+  }
```
Comment (Panquesito7) [inline public/data/dunyanong.json:18]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by emmalearnscode — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]

**Review by vikasganiga05 — APPROVED** [review]
LGTM :+1:

File: public/data/dunyanong.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        "url": "https://github.com/dunyanong",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/dun_yan_",
+        "icon": "twitter"
+      }
+    ]
+}
```
Comment (Panquesito7) [inline public/data/dunyanong.json:18]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by krshkodes — APPROVED** [review]
Looks awesome! Endline to be handled by bot!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to me!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

**Pull request merged**